src/popup/components/TransactionOverview.vue

```diff
@@ -100,7 +100,27 @@ export default defineComponent({
         callerId,
       } = innerTx.value;
 
-      switch (outerTxTag.value) {
+      if (outerTxTag.value === Tag.PayingForTx && innerTxTag.value === Tag.GaAttachTx) {
+        return {
+          sender: {
+            address: innerTx.value.ownerId,
+            name: getName(innerTx.value.ownerId).value,
+            url: protocolExplorer.prepareUrlForAccount(innerTx.value.ownerId),
+            label: t('multisig.multisigVault'),
+          },
+          recipient: {
+            label: t('common.smartContract'),
+            address: innerTx.value.contractId,
+          },
+        };
+      }
+
+      const transactionTag = (
+        outerTxTag.value === Tag.PayingForTx
+        || outerTxTag.value === Tag.GaMetaTx
+      ) ? innerTxTag.value : outerTxTag.value;
```

Comment:
Maybe you can use `getTxTag` utility function?

Reply:
`innerTxTag` is a `getTxTag` of `innerTx` and `outerTxTag` is `getTxTag` of `outerTx`. And `getTxTag` function will not return us tag that is required for the overview.